spec/rspec/rails/configuration_spec.rb

```diff
@@ -257,8 +257,8 @@ def self.application; end
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
 
-    describe 'leans test mailbox after each example' do
-      let(:base_mailer) do
+    describe 'clears ActionMailer::Base::Deliveries after each example' do
```

Comment:
Sorry for the late appearance. Do you think it makes sense to add `order: :defined` here?

Reply:
Their is no need. You need at least two example, order does not mater. :)

Reply:
Yep they are order independant, they just need different emails.

Reply:
👍